packages/ffe-icons/icons/pil-hoyre-sirkel.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 39 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 19.5C0 8.73045 8.73045 0 19.5 0C30.2696 0 39 8.73045 39 19.5V20.5C39 31.2696 30.2696 40 19.5 40C8.73045 40 0 31.2696 0 20.5V19.5Z"/>
+<path d="M10.8961 21.6H24.5761C22.9561 22.7 21.7361 24.46 21.2761 26.12L22.6961 26.64C23.4961 24.28 25.2361 21.6 28.0961 21.6V20.18C25.2361 20.18 23.4961 17.5 22.6961 15.14L21.2761 15.66C21.7361 17.34 22.9761 19.1 24.5961 20.2H10.8961V21.6Z" fill="white"/>
```

Comment:
Litt usikker på om vi skal ha fill her?  Det kan kanskje @antidecaf  svare bedre på enn meg. Fant ingen andre eksempler med to path elementer i svgen, for å se hvordan andre løste det.

Reply:
Dersom jeg fjerner fill, vil pilen bli gjennomsiktig. Er det noe vi vil?

Reply:
I dette tilfellet ville det kanskje gitt mening å slå pathene sammen, slik at selve pila er transparent.